Doc/library/socket.rst

```diff
@@ -595,6 +595,21 @@ The following functions all create :ref:`socket objects <socket-objects>`.
    .. versionchanged:: 3.2
       *source_address* was added.
 
+.. function:: bind_socket(address, family=AF_UNSPEC, type=SOCK_STREAM, *,
```

Comment:
Would you consider making all arguments except `address` keyword-only?

Reply:
I changed my mind about this (and updated code).